### PR TITLE
Move WebRTC Endpoint to another package

### PR DESCRIPTION
**Pull request by czerwiukk**
## Description

WebRTC endpoint can be used without FishjamClient to work with Membrane RTC Engine alone.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)

### CODE REVIEW

File: packages/webrtc-client/package.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@fishjam-cloud/webrtc-client",
+  "version": "0.9.0",
```
Comment (roznawsk) [inline packages/webrtc-client/package.json:3]: Maybe we should start versioning from `0.1.0`

File: packages/webrtc-client/package.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  "description": "Typescript client library for Fishjam Cloud",
```
Comment (roznawsk) [inline packages/webrtc-client/package.json:4]:
```suggestion
  "description": "Typescript client library for ExWebRTC/WebRTC endpoint in Membrane RTC Engine",
```

**Review by roznawsk — APPROVED** [review]
LGTM

File: packages/webrtc-client/package.json (hunk 1 above)
Comment (czerwiukk) [reply packages/webrtc-client/package.json:3]: why should we if the whole monorepo shares the same version?

**Pull request merged**